commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/arguments/ListArgumentBuilder.java

```diff
@@ -129,6 +141,9 @@ public ListArgumentBuilderFinished withStringMapper() {
 		 * @return this list argument builder
 		 */
 		public ListArgumentBuilderFinished withMapper(Function<T, String> mapper) {
+			if (allowAnyValue && !(mapper instanceof StringTooltip)){
+				throw new IllegalArgumentException("allowAnyValue cannot be used with non-string mappers");
```

Comment:
I'd say this should be something like:
`allowAnyValue can only be set to true if a String mapper is used.`

However, I do not know if that even is better so maybe we could wait for other opinions too.